scheduler/scheduler_balance.go

```diff
@@ -79,7 +78,6 @@ func (b *balanceScheduler) Schedule(
 		b.random, aliveCaptures, stateMachines, batchSize)
 	b.forceBalance = len(tasks) != 0
 	log.Info("balance scheduler generate tasks",
-		zap.String("id", b.id.String()),
```

Comment:
This field is used to distinguish between different schedulers.

Reply:
I will add it back